src/color/base.json

```diff
@@ -35,17 +35,34 @@
           "public": true
         },
         "300": {
-          "value": "b2b2b2"
+          "value": "b2b2b2",
+          "public": false
```

Comment:
This flag is not needed. It only needs to be added when the value is to be `true`. `false` is implied.